client/my-sites/plan-features-2023-grid/lib/test/sort-plan-properties.ts

```diff
@@ -12,27 +12,27 @@ describe( 'sortPlans', () => {
 	const planFree = {
 		rawPrice: 0,
 		planName: 'free',
-	} as PlanProperties;
+	} as unknown as PlanProperties;
 
 	const planPersonal = {
 		rawPrice: 100,
 		planName: 'personal',
-	} as PlanProperties;
+	} as unknown as PlanProperties;
 
 	const planPremium = {
 		rawPrice: 200,
 		planName: 'premium',
-	} as PlanProperties;
+	} as unknown as PlanProperties;
```

Comment:
Hey @chriskmnds I am unfamiliar with this approach. What is it trying to acheive? 🤔